### ISSUE TITLE
The content of amyloidPET_FBP_TP0 dataset is strange? 

### ISSUE THREAD

**Issue by wangxiao5791509** [NiftyPET/NIPET#44]
[https://niftypet.readthedocs.io/en/latest/tutorials/demo/](https://niftypet.readthedocs.io/en/latest/tutorials/demo/) 
{'corepath': 'amyloidPET_FBP_TP0',
 'lm_dcm': 'amyloidPET_FBP_TP0/LM/17598013_1946_20150604155500.000000.dcm',
 'lm_bf': 'amyloidPET_FBP_TP0/LM/17598013_1946_20150604155500.000000.bf',
 'hmumap': 'amyloidPET_FBP_TP0/niftyout/mumap-hdw/hmumap.npz',
 'sinos': 'amyloidPET_FBP_TP0/niftyout/sino/sinos_s11_frm-0-0.npz',
 'nrm_dcm': 'amyloidPET_FBP_TP0/norm/17598013_1946_20150604082431.000000.dcm',
 'nrm_bf': 'amyloidPET_FBP_TP0/norm/17598013_1946_20150604082431.000000.bf',
 'pCT': 'amyloidPET_FBP_TP0/pCT/17598013_pCT_synth.nii.gz',
 'T1lbl': 'amyloidPET_FBP_TP0/prcl/n4_bias_corrected_T1_NeuroMorph_Parcellation.nii.gz',
 'T1nii': 'amyloidPET_FBP_TP0/T1w_N4/t1_S00113_17598013_N4bias_cut.nii.gz',
 'T1N4': 'amyloidPET_FBP_TP0/T1w_N4/t1_S00113_17598013_N4bias_cut.nii.gz',
 'mumapDCM': 'amyloidPET_FBP_TP0/umap',
 '#mumapDCM': 192}

bu the content of amyloidPET_FBP_TP0 dataset I downloaded are: 
{'corepath': '/home/wangxiao/Downloads/projects/PET_reconstuction/NIPET/dataset/amyloidPET_FBP_TP0/', 
'lm_dcm': '/home/wangxiao/Downloads/projects/PET_reconstuction/NIPET/dataset/amyloidPET_FBP_TP0/LM/17598013_1946_20150604155500.000000.dcm', 
'lm_bf': '/home/wangxiao/Downloads/projects/PET_reconstuction/NIPET/dataset/amyloidPET_FBP_TP0/LM/17598013_1946_20150604155500.000000.bf', 
'mumapDCM': '/home/wangxiao/Downloads/projects/PET_reconstuction/NIPET/dataset/amyloidPET_FBP_TP0/umap', 
'#mumapDCM': 192, 
'nrm_dcm': '/home/wangxiao/Downloads/projects/PET_reconstuction/NIPET/dataset/amyloidPET_FBP_TP0/norm/17598013_1946_20150604082431.000000.dcm', 
'nrm_bf': '/home/wangxiao/Downloads/projects/PET_reconstuction/NIPET/dataset/amyloidPET_FBP_TP0/norm/17598013_1946_20150604082431.000000.bf'}

**Comment by yunghx** [comment]
You are right, some files are missing, and I am also not able to follow the demo in the referred page....
